samples/config_getting_started_segmentation_rad3d.yaml

```diff
@@ -89,6 +89,6 @@ scheduler:
 track_memory_usage: false
 verbose: true
 version:
-  maximum: 0.0.20
-  minimum: 0.0.20
+  maximum: 0.1.0-dev
```

Comment:
general comment regarding all configs with the min/max version specs - I do not know how it is done now, but if manually then perhaps some simple script doing grep and sed operations could do.

Reply:
We have it: https://github.com/mlcommons/GaNDLF/blob/new-apis_v0.1.0-dev/update_version.py